### PR TITLE
logger: forward the loglevel thru to the web client

### PR DESCRIPTION
**Pull request by nicks**
Hello @landism, @hyu,

Please review the following commits I made in branch nicks/level:

d920d9861f0b953910a39d228113ce8788a145c7 (2019-12-10 18:17:45 -0500)
logger: forward the loglevel thru to the web client
eventually the webclient will be able to dynamically change the log level

this just gets us towards that world by shipping the right data around

### CODE REVIEW

**Comment by landism** [comment]
(sorry, reviewed this yesterday morning and apparently failed to hit publish)

File: pkg/model/logstore/logstore.go [hunk 1]
```diff
@@ -458,6 +471,10 @@ func (s *LogStore) ManifestLog(mn model.ManifestName) string {
 				continue
 			}
 
+			if !currentSeg.CanContinueLine(segment) {
+				break
```
[the remark this answers is not in the document]
Comment (nicks) [reply pkg/model/logstore/logstore.go]: good question! when the logs are appended, we'll correctly mark up the line continuations. i added your example as a test to confirm this.

re: LogInterleaving. Ya, that's right. This is really more of a product decision than a technical one, but the intuition is that:
- All the logs from a single source should appear in the order that they're printed, because that's what the source intended to happen.
- If Tilt is interleaving logs from different sources, it should try to re-order them to preserve the integrity of each source.

In other words: if two different pod logs are getting interleaved by Tilt, then that's the LogStore's problem to sort out. Because who else would fix it? The individual pods can't do anything to fix it. 

If, say, there's a bug in Tilt's ImageBuilder where it's interleaving Info and Debug and Warning logs in a weird way, then we should just fix BuildController.
